serialization.go

```diff
@@ -314,15 +332,17 @@ func InferAcceptHeaderFromType(ans any) string {
 	return "application/json"
 }
 
-func parseAcceptHeader(accept string, ans any) string {
-	if strings.Index(accept, ",") > 0 {
-		accept = accept[:strings.Index(accept, ",")]
-	}
-	if accept == "*/*" {
-		accept = ""
+func inferAcceptHeader(accept string, ans any) string {
+	if accept == "" || accept == "*/*" {
+		return InferAcceptHeaderFromType(ans)
 	}
+	return accept
+}
+
+func parseAcceptHeader(header http.Header) []string {
+	accept := header.Get("Accept")
 	if accept == "" {
-		accept = InferAcceptHeaderFromType(ans)
+		return []string{""}
```

Comment:
With this behaviour, if no Accept header is sent, nothing will be sent. I suggest to provide application/json in that case, what do you think?

Reply:
Take a look at `inferAcceptHeader`. Here is the test case: https://github.com/go-fuego/fuego/pull/175/files/60bf483061906c646f99dfb4ac7c25a6c67378a7#diff-95e7bbc519ba50d88c1639d2f79e997f1011f802b8c69605813150752b40c535R470